spark/src/main/scala/io/substrait/spark/logical/ToSubstraitRel.scala

```diff
@@ -492,6 +534,8 @@ private[logical] class WithLogicalSubQuery(toSubstraitRel: ToSubstraitRel)
   override protected val toScalarFunction: ToScalarFunction =
     ToScalarFunction(SparkExtension.SparkScalarFunctions)
 
+  val existenceJoins = scala.collection.mutable.Map[Long, SExpression.InPredicate]()
```

Comment:
make this `private val` and move to the top of the file?

Reply:
Sure.  I've also added a public lookup method so that the `translateAttribute` method can still access it.